indexer/services/ender/src/scripts/helpers/dydx_liquidation_fill_handler_per_order.sql

```diff
@@ -200,7 +200,7 @@ BEGIN
                 perpetual_position_record."side", order_side) THEN
             sum_open = dydx_trim_scale(perpetual_position_record."sumOpen" + fill_amount);
             entry_price = dydx_get_weighted_average(
-                    perpetual_position_record."entryPrice", perpetual_position_record."sumOpen",
+                    perpetual_position_record."entryPrice", ABS(perpetual_position_record."size"),
```

Comment:
nit: Possible to add a test-case where perpetual position size is = negative size of fill to test that the previous error case couldn't happen again?